parlai/tasks/flattened/agents.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""Generates a flattened version of a ParlAI task, i.e., for tasks with
+multi-turn dialogues (episodes), this will generate a task with single
+example episodes, in which the context from previous dialogue turns in each
+episode are included in the 'text' field of each example.
+
+In order to use this teacher, specify the task flag as follows:
+`--task flattened:task:<ORIGINAL TASK NAME>`.
```

Comment:
perhaps we can just do `flattened:<TASK NAME>` to make it shorter? or is `task:...` necessary for accessing the `TaskTeacher` below?